packaging/smoke_test.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+def main():
+    """
+    Run torchao binary smoke tests. This is a placeholder for future tests. See
+    https://github.com/pytorch/test-infra/wiki/Using-Nova-Reusable-Build-Workflows
+    for more information about Nova binary workflow.
+    """
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
We can just run `pytest .` or `pytest/test/test_ops` since ops are what's most likely to break

Reply:
We sure can, let me add that and see how it goes

Reply:
It's easier to shell out pytest command in `packaging/post_build_script.sh`, so let me add it there instead.  The python `smoke_test.py` could still be used to perform simple operations to validate.